es-data.yaml

```diff
@@ -56,7 +58,11 @@ spec:
         volumeMounts:
         - name: storage
           mountPath: /data
-      volumes:
-          - emptyDir:
-              medium: ""
-            name: "storage"
+  volumeClaimTemplates:
+  - metadata:
+      name: storage
+    spec:
+      accessModes: [ "ReadWriteOnce" ]
+      resources:
+        requests:
+          storage: 12Gi
```

Comment:
Hey sorry , been maxed out, so I just wanted to show how you can use a volume e.g. in my case EBS volume attached to the pod for data , I notice now my comment was a bit sparse... I spent some time trying to use persistent volumes , but the naming causes problems , unless there is a trick I don't know with stateful sets I don't know to get them to choose them (still on a learning curve with k8s)?